txn_writer.go

```diff
@@ -21,15 +21,63 @@ import (
 // Example string to be written to the manifest file
 // if no dependencies are found in the project
 // during `dep init`
-const exampleToml = `
-# Example:
+const exampleTOML = `
+## EXAMPLE (these lines may be deleted)
+#
+## "required" lists a set of packages (not projects) that must be included in
+## Gopkg.lock. This list is merged with the set of packages imported by the current
+## project. Use it when your project needs a package it doesn't explicitly import -
+## including "main" packages.
+# required = ["github.com/user/thing/cmd/thing"]
+#
```

Comment:
Heh, I introduced these to try to visually delineate between the groups of items, but now that their ordering is actually correct, can just rely on...yknow, TOML's rules 😄